web/client/src/library/components/editor/EditorFooter.tsx

```diff
@@ -73,7 +73,7 @@ export default function EditorFooter({ tab }: { tab: EditorTab }): JSX.Element {
           </EditorIndicator.Selector>
         </EditorIndicator>
       )}
-      {tab.file.isSQLMeshModel && tab.dialect != null && (
+      {tab.file.isSQLMeshModel && tab.dialect != null && tab.dialect !== '' && (
```

Comment:
Can you do a truthy check here?
```suggestion
      {tab.file.isSQLMeshModel && tab.dialect && (
```

Reply:
Our Ts config set that we need to explicitly do `tab.dialect != null` to tell TS that `tab` exits for all code below